sdks/go/pkg/beam/core/sdf/watermarkEstimator.go

```diff
@@ -0,0 +1,24 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
WRT the file name, prefer snake_case over lowerCamelCase.

Or simplify the name to estimators.go or watermarks.go and avoid the question entirely.

Reply:
I went with snake case